spec/system/checklist_items/destroy_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "checklist_items/destroy", type: :system do
+  let(:casa_admin) { create(:casa_admin) }
+  let(:hearing_type) { create(:hearing_type) }
+  let(:checklist_item) { create(:checklist_item) }
+
+  before do
+    hearing_type.checklist_items.push(checklist_item)
```

Comment:
I tried to set the checklist item's hearing type the same way that the edit spec is set up, but it didn't work. This is what I came up with. If there's a better way, I'm open to ideas, because this smells to me.

Reply:
I figured out a better way.